### DISCUSSION TITLE
Is this caused by this software "Eject NVIDIA GeForce RTX .. Laptop GPU"

### DISCUSSION THREAD

**Discussion by talhazengin** [seerge/g-helper#1370] [General]
Windows shows this "Eject NVIDIA GeForce RTX .. Laptop GPU" on quick toolbar. How can i remove this suggestion?

**Comment by seerge** [comment]
@talhazengin hello,

This is how Windows reacts on suddenly appearing new device (your GPU) when you enable it by setting Standard mode after Eco.

Clicking on actual "Eject" will do absolutely nothing (it won't even try to disable device). This is rather strange / funny side effect of how windows works (it literally thinks it's something you have plugged in into computer, like a usb-drive)

**Reply by talhazengin** [reply]
:) Thank you for this nice information.

**Reply by m31-galaxy** [reply]
Hi, I'm experiencing this issue as well. Tried ejecting the GPU and to my surprise it did actually work.

![image](https://github.com/seerge/g-helper/assets/26030336/f802df12-89d9-4ee0-ae2a-6c37e9ac7420)
![image](https://github.com/seerge/g-helper/assets/26030336/52f91574-978d-4697-8943-621f15756c71)

**Comment by pyunjiji** [comment]
It will probably disappear if you reinstall the asus touchpad driver.

**Reply by Kikimahe** [reply]
It's a good fix if windows want always to eject GPU after a Windows clean install ;-)

**Reply by talhazengin** [reply]
It was a really interesting solution, thank you.

**Reply by Scarlaid** [reply]
I reinstalled recently and it still show, as a ghost driver...
![image](https://github.com/seerge/g-helper/assets/90851437/dac7122f-ec70-436c-934f-93fb6da91242)

**Reply by Kikimahe** [reply]
@Scarlaid 
Try to install  "ASUS Precision Touchpad driver" and restart windows.
If the GPU is responsible, this will fix it.

**Reply by Scarlaid** [reply]
@Kikimahe I installed it, and the icon is still present
I have also tried to extract the installer and install it through its bat file, still present

**Reply by AudreyAP** [reply]
I also have this, even after installing the touchpad driver. Touchpad driver makes the GPU line go away but replaces it with dots.

**Reply by Kikimahe** [reply]
Look in the “Taskbar/Other System Tray Icons” to see if you have another icon that looks like the eject icon.
J'ai déja eu plusieurs icones identiques dans ce menu.

for example
![Capture d'écran 2023-10-02 075430](https://github.com/seerge/g-helper/assets/142881210/a10521e3-666c-4894-93aa-58aeb3d76dd2)


You can also try to uninstall and clean the GPU driver with the "DDU-Display driver Uninstaller" tool. 
Then reinstall :
- The GPU driver
- ASUS Precision Touchpad driver

**Reply by NakeNoob** [reply]
> @Scarlaid Try to install "ASUS Precision Touchpad driver" and restart windows. If the GPU is responsible, this will fix it.
WORKS FOR ME !!!
 Asus Tuf A15 (2023)
 after install "ASUS Precision Touchpad driver" and restart
 Thank you <3

**Comment by Scarlaid** [comment]
`reg add HKCU\Software\Microsoft\Windows\CurrentVersion\Applets\SysTray /v “Services” /t reg_dword /d 29 /f systray`

Just found an alternative solution. If it annoys you, best to just remove it by running the above 👌

**Reply by AudreyAP** [reply]
Doesn't this disable the icon entirely? So, no more ejecting?

**Reply by Scarlaid** [reply]
thats the point 🤯

**Reply by AudreyAP** [reply]
@Scarlaid how do I eject usb devices then?

**Reply by Saaaiiif** [reply]
I know this is late but, you can eject usb devices by right clicking it in the file explorer and choosing eject.

**Comment by m31-galaxy** [comment]
An update to this, it appears that for me this only occurs if I'm using Eco mode and switch to Standard. If I boot straight into Standard mode then there is no such "ghost device".

**Comment by Bergoglio87** [comment]
April 2025 now. I have an ASUS ROG with the RTX4070M.
### Issue resolved by installing the "ASUS Precision TouchPad Driver".
Original source: https://www.youtube.com/watch?v=O7m3nKvtWAg
Thanks! Cheers ^
